File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main()
{
    // Prompt user for half-pyramid's height (non-negative and no greater than 23)
    int height;
    do
    {
        printf("height: ");
        height = GetInt();
    } 
    while (height < 0 || height > 23);
    
    // Generate and draw the half-pyramid
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k <= i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario.py
import cs50

def main():
    while True:
        print("Height: ", end = "")
        height = cs50.get_int()
        if height >= 0 and height <= 23:
            break
        
    for i in range(height):
        for j in range(height - i - 1):
            print(" ", end = "")
        for k in range(i+2):
            print("#", end = "")
        print("")
            
if __name__ == "__main__":
    main()